templates/flavors/README.md

```diff
@@ -14,3 +14,77 @@ appended to the end of the cluster-template.yaml, e.g cluster-template-{director
 flavor can be used by specifying `--flavor {directory-name}`.
 
 To generate all CAPZ flavors, run `make generate-flavors`.
+
+
+## Running flavor clusters as a tilt resource
+
+#### From CLI
+run ```tilt up ${flavors}``` to spin up worker clusters in azure represented by a tilt local resource.  You can also modify flavors while tilt is up by running ```tilt args ${flavors}```
+
+#### From Tilt Config
+Add your desired flavors to tilt_config.json:
+```json
+{
+    "worker-flavors": ["default", "aks external-cloud-provider", "machinepool", "system-assigned-identity", "user-assigned-identity"]
+}
+```
+
+#### Requirements
+Please note your tilt-settings.json must contain at minimum the following fields:
+```json
+{
+    "kustomize_substitutions": {
```

Comment:
Isn't the minimum actually just the first 4 like documented in https://github.com/kubernetes-sigs/cluster-api-provider-azure/blob/master/docs/development.md#tilt-for-dev-in-capz?

```
{
  "kustomize_substitutions": {
        "AZURE_SUBSCRIPTION_ID_B64": "******",
        "AZURE_TENANT_ID_B64": "******",
        "AZURE_CLIENT_SECRET_B64": "******",
        "AZURE_CLIENT_ID_B64": "******",
  }
}
```

Reply:
Or is that a different requirement just when using tilt + flavors? If so, let's make that explicit, something like :

```
Please note your tilt-settings.json must contain at minimum the following fields when using tilt resources to deploy cluster flavors :
```

Reply:
Yes, the non B64 ones are only required when using flavors.  Good idea-- I will add that!